index.js

```diff
@@ -68,7 +68,7 @@ function uglifyify(file, opts) {
     if (debug && matched) {
       opts.sourceMap.content = convert.fromJSON(
         new Buffer(matched[1], 'base64').toString()
-      ).sourceMap
+      ).sourcemap
```

Comment:
I think this needs to be [camel-cased now](https://github.com/mishoo/UglifyJS2/tree/harmony#source-map-options), unless you have different information from somewhere else?